config/kernel.m4

```diff
@@ -515,13 +515,39 @@ AC_DEFUN([ZFS_AC_KERNEL], [
 
 	AC_MSG_RESULT([$kernsrcver])
 
-	AS_VERSION_COMPARE([$kernsrcver], [$ZFS_META_KVER_MIN], [
-		 AC_MSG_ERROR([
+	AX_COMPARE_VERSION([$kernsrcver], [ge], [$ZFS_META_KVER_MIN], [], [
+		AC_MSG_ERROR([
```

Comment:
Now that the pre-4.18 kernel changes have been merged it look like there's only one place left which uses `AS_VERSION_COMPARE`.  Once this merges we should switch the `AS_VERSION_COMPARE` call in `config/kernel-blkdev.m4` over to `AX_COMPARE_VERSION`.

Reply:
Oh yeah, good spot. Noted.